store/AuthContext.js

```diff
@@ -45,18 +45,29 @@ export function AuthProvider({ children }) {
           emailVerified: user.emailVerified,
           accountType: user.photoURL,
         }
-        const token = await user.getIdToken(true)
-
+        const token = await user.getIdToken()
         // fetchUserInfo(userObject)
         setCurrentUser(userObject)
-        nookies.set(undefined, 'token', token, {})
+        nookies.set(undefined, 'token', token, { path: '/' })
```

Comment:
setting the path explicitly ensures that the cookie has the same path every time it's set. token cookies were inadvertently being set twice at times with different paths, which meant expired tokens were hanging around and not getting deleted when calling `nookies.destroy`